### PR TITLE
chore: restructure tutorials

### PR DESCRIPTION
**Pull request by mihai-sysbio**
### Main improvements in this PR:
This PR aims to restructure the tutorials, resolving #308.

In short, it establishes self-standing, independent tutorials (with all subfolders like `userData` before, including the adapter). Each tutorial also has now its own protocol.
The way the adapters are handles has been slightly tweaked.

Expected to be restructured:
- [x] tutorial for yeast-GEM:
  - [x] stuck as step 23, reported independently [here](https://github.com/SysBioChalmers/GECKO/issues/305))
  - [x] should be the full `protocol.m`, with all comments
- [ ] tutorial for Human-GEM
  - [ ] using `geckoLight` variant
- [ ] tutorial for E.coli (@Yu-sysbio is working on this)
- [ ] ~blank example~
  - [ ] ~with generic `protocol.m` and `adapter`~

**I hereby confirm that I have:**

- [x] Selected `develop` as a target branch (top left drop-down menu)

### CODE REVIEW

**Comment by ae-tafur** [comment]
I think that adding the word tutorial as a prefix inside the tutorial folder may be redundant, isn't it ? or is there a specific reason to do this ?

**Comment by mihai-sysbio** [comment]
> I think that adding the word tutorial as a prefix inside the `tutorial` folder may be redundant, isn't it ? or is there a specific reason to do this ?

Fair point, I also hesitated about this. My reasoning was that people will likely copy one of these folders, so when they do it will be clearer what is the "tutorial" what is what they modified. I might be wrong here, we should follow what is most intuitive for the users so it's good to have other opinions.

File: test/unit_tests/ecTestGEM/TestGEMAdapter.m [hunk 1]
```diff
@@ -5,8 +5,8 @@
             
             %Directory where all model-specific files and scripts are kept.
             %Is assumed to follow the GECKO-defined folder structure. The
-            %code below refers to userData/ecYeastGEM in the GECKO path.
-            obj.params.path = fullfile(findGECKOroot,'test','unit_tests','ecTestGEM');
+            %code below refers to tutorials/tutorial_yeast-GEM in the GECKO path.
+            obj.params.path = fullfile(findGECKOroot,'test','unit_tests','ecTestGEM', 'TestGEMAdapter.m');
```
Comment (mihai-sysbio) [inline test/unit_tests/ecTestGEM/TestGEMAdapter.m:9]: The path was mistakenly altered here:
```suggestion
            obj.params.path = fullfile(findGECKOroot,'test','unit_tests','ecTestGEM');
```

Comment (mihai-sysbio) [reply test/unit_tests/ecTestGEM/TestGEMAdapter.m]: I'm confirming here the fix worked.

**Comment by edkerk** [comment]
Recent changes:
- Remove `blankExample`, as it is replaced by `startGECKOproject` (see #312)
- Restore (and update) comments in ecYeastGEM `protocol.m` (as agreed upon in #308)
- Run through the ecYeastGEM protocol
- Simplified `ModelAdapterManager` commands (most relevant: `.getDefault` instead of `.getDefaultAdapterFromPath`)

Remaining:
- `ecHumanGEM` tutorial with GECKO light
- `ecEcoli` tutorial

I suggest to have these addressed in separate PRs, so that this PR with its significant refactoring of file locations can be merged with `develop`. I will open two Issues for the remaining tasks.

**Review by edkerk — APPROVED** [review]

**Pull request merged**